Library/Application Support/Code/User/keybindings.json

```diff
@@ -140,9 +132,8 @@
     },
     {
         "key": "cmd+i",
-        "command": "references-view.findImplementations"
+        "command": "editor.action.goToImplementation"
```

Comment:
Use goToImplementation rather than findImplementations to avoid opening up the sidebar and to make it inline with gotoReferences